test/src/test/kotlin/graphql/nadel/tests/next/fixtures/hydration/copy/HydrationCopiesFieldAndHasPolymorphicHydrationTest.kt

```diff
@@ -0,0 +1,298 @@
+
+                                ids!!
+                                    .map {
+                                        issuesByIds[it]
+                                    }
+                            }
+                    }
+            },
+        ),
+        Service(
+            name = "bitbucket",
+            overallSchema = """
+                type Query {
+                  pullRequestsByIds(ids: [ID!]!): [BitbucketPullRequest]
+                }
+                type BitbucketPullRequest {
+                  id: ID!
+                  title: String
+                  patch: String
+                }
+            """.trimIndent(),
+            runtimeWiring = { runtime ->
+                data class BitbucketPullRequest(
+                    val id: String,
+                    val title: String,
+                    val patch: String,
+                )
+
+                val issuesByIds = listOf(
+                    BitbucketPullRequest(
+                        id = "ari:cloud:bitbucket::pull-request/1",
+                        title = "Delete everything",
+                        patch = "-",
+                    ),
+                    BitbucketPullRequest(
+                        id = "ari:cloud:bitbucket::pull-request/2",
+                        title = "Initial Commit",
+                        patch = "+",
+                    ),
+                ).strictAssociateBy { it.id }
+
+                runtime
+                    .type("Query") { type ->
+                        type
+                            .dataFetcher("pullRequestsByIds") { env ->
+                                val ids = env.getArgument<List<String>>("ids")
+
+                                ids!!
+                                    .map {
+                                        issuesByIds[it]
+                                    }
+                            }
+                    }
+            },
+        ),
+        // Service that introduces virtual type
+        Service(
+            name = "work",
+            overallSchema = """
+                type Query {
+                  businessReport_findRecentWorkByTeam(
+                    teamId: ID!
+                    first: Int
+                    after: String
+                  ): WorkConnection
+                    @hydrated(
+                      service: "graph_store",
+                      field: "graphStore_query"
+                      arguments: [
+                        {
+                          name: "query"
+                          value: "SELECT * FROM Work WHERE teamId = ?"
+                        }
+                        {
+                          name: "first"
+                          value: "$argument.first"
+                        }
+                        {
+                          name: "after"
+                          value: "$argument.after"
+                        }
+                      ]
+                    )
+                }
```

Comment:
Does this code have a "get all other parameters" and stuff them into a map code yet?

Reply:
Not yet, coming in another PR.

Branch is https://github.com/atlassian-labs/nadel/tree/remaining-arguments